### ISSUE TITLE
GitHub "Run workflow/Build and Push Docker Image" : unauthorized: Application not registered with AAD

### ISSUE THREAD

**Issue by rajredd** [microsoft/MCW-Cloud-native-applications#203]
I am getting an error during "Run workflows" in github. (Exercise 1/Task 8/Step 16) (see the attached screen shot), can you help?

Task 8: Setup CI Pipeline to Push Images

16. Selecting the Build and Push Docker Image job of the workflow will display it's execution status.

![image](https://user-images.githubusercontent.com/37375319/99150845-4ec4ba00-2665-11eb-8972-dfa5bcc69c60.png)

**Comment by rajredd** [comment]
Fabmedical ACR has a Service Principle created as instructed in "Before HOL". Not sure where the authorization to ACR took place during this GitHub action of running workflow?

**Comment by rajredd** [comment]
Sorry, I entered the ACR_USERNAME of the secret without the suffix. I fixed it now. the login worked. but i am getting this error

unable to prepare context: path "./content-web" not found; can you tell where "./content-web" was expected? in "build agent" or ACR repository? I am confused. 

![image](https://user-images.githubusercontent.com/37375319/99151601-69e5f880-266a-11eb-81ce-91de8203e83f.png)

**Comment by saimachi** [comment]
Hello @rajredd,

When we executed the lab, we were unable to isolate the same issue. As for your question, "./content-web" refers to the location within your Fabmedical GitHub repository. If you look at the content-web.yml workflow file, that path and the Dockerfile present within that location are used to build a container that is pushed to Azure CR.

I took a look at your repository at https://github.com/rajredd/Fabmedical. It looks like you forked the Microsoft Cloud Native lab repository. However, the Fabmedical repository must be initialized with the content-web, content-init, and content-api directories, as mentioned in Task 7 of the Before the Hands-on Lab instruction document (https://github.com/microsoft/MCW-Cloud-native-applications/blob/master/Hands-on%20lab/Before%20the%20HOL%20-%20Cloud-native%20applications.md#task-7-create-a-github-repository).

If you want a reference, here is a copy of my Fabmedical repository: https://github.com/saimachi/Fabmedical.

**Comment by rajredd** [comment]
Thank you for your reply. I was very sure that I followed the "Before HOL" instructions. Now I realized that I was following https://stmsdaysprerequisites.blob.core.windows.net/ms-days-prerequisites/Before%20the%20HOL%20-%20Cloud-native%20applications.pdf
this instruction was given from an online workshop offered last week that I was not able to attend. I was trying to do the lab myself after the online workshop was over. In the above instruction, task 7 was different from task 7 you mentioned

Task 7: Setup Azure DevOps project

I did not perform 

Task 7: Create a GitHub repository.
